swap_meet/clothing.py

```diff
@@ -1,2 +1,17 @@
-class Clothing:
-    pass
\ No newline at end of file
+from uuid import uuid4 
+from .item import Item
+
+class Clothing(Item): # Updated class to inherit from Item
+    def __init__(self, id=None, fabric="Unknown", condition=0):
+        self.id = uuid4().int if id is None else id
+        self.fabric = fabric if fabric != "Unknown" else "Unknown"
+        super().__init__(id, condition)  # pass condition to Item
+
+    def get_category(self):
+        return self.__class__.__name__
+    
+    def __str__(self):
+        return f"An object of type Clothing with id {self.id}."\
+        f" It is made from {self.fabric} fabric."
```

Comment:
The first sentence in the string you return looks __very__ similar to what the `__str__` method in `Item` returns. 

If you refactor `__str__` in the `Item` class so that you don't hardcode the word "item" in that string, then you can reuse the functionality here. 

```suggestion
        return f"{super().__str__()} It is made from {self.fabric} fabric."
```

Reply:
Thank you for this suggestion! Modified __str__ in parent and child modules.